login.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/login.css">
+    <title>login</title>
+</head>
+<body>
+    <div class="login_box">
+        <div class="logo_img"><a href="/index.html"><img src="img/lg.png" alt="판다마켓"></a></div>
+        <div class="login_text">
+            <p class="login_tx">이메일</p>
+            <input class="login_input" type="text" placeholder="이메일을 입력해주세요">
+            <p class="login_tx">비밀번호</p>
+            <div class="input_wrapper">
+                <input class="login_input password" type="password" placeholder="비밀번호를 입력해주세요">
+                <img class="toggle_pw" src="img/btn_visibility_off.png" alt="비밀번호 보기">
+            </div>
+            <button class="login_btn">로그인</button>
```

Comment:
💊 제안
button 태그의 type 속성을 명시하지 않으면 submit을 기본값으로 가지게 됩니다~
이러한 type 속성은 button 태그에 중요한 요소이므로 명시적으로 작성해주는 것을 추천드려요.

```suggestion
            <button type="button" class="login_btn">로그인</button>
```

추가로 type이 submit인 button의 경우 form에서 제출 이벤트가 발생하면, 해당 요소의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.

```html
// case1: form에 onsubmit 이벤트 명시
<form class="login-form" onsubmit="login">
  <input /> // input에서 enter 키 입력시 onsubmit 이벤트 실행
  <button type="submit">login</button> // 해당 버튼 클릭 시 onsubmit 이벤트 실행
</form>

// case2: submit 버튼에 onclick 이벤트 명시
<form class="login-form">
  <input /> // input에서 enter 키 입력시 button의 onclick 이벤트 실행
  <button type="submit" onclick="login">login</button>
</form>
```

위에서처럼 동작이 될 수 있도록 form 태그도 추가해보세요!

[mdn button](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button#type)